hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/CompactionCommand.java

```diff
@@ -54,6 +55,27 @@
 
   private static Logger log = LogManager.getLogger(HDFSParquetImportCommand.class);
 
+  @CliAvailabilityIndicator({"compactions show all"})
+  public boolean isAvailable() {
+    return (HoodieCLI.tableMetadata != null)
+        && (HoodieCLI.tableMetadata.getTableType() == HoodieTableType.MERGE_ON_READ);
+  }
+
+  @CliAvailabilityIndicator({"compaction show"})
```

Comment:
What's the difference between this and the one above ?

Reply:
Good question :)  Realized this can be made simple .  Fixed.